core/web/job_specs_controller.go

```diff
@@ -45,6 +57,8 @@ func (jsc *JobSpecsController) Create(c *gin.Context) {
 	if err := c.ShouldBindJSON(&jsr); err != nil {
 		jsonAPIError(c, http.StatusBadRequest, err)
 	} else if js := models.NewJobFromRequest(jsr); false {
+	} else if err := jsc.requireAllowed(js); err != nil {
+		jsonAPIError(c, http.StatusBadRequest, err)
```

Comment:
This might make more sense https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/501

Reply:
Makes sense, done :+1: